architecture-building-systems/CityEnergyAnalyst
# 2745 port codebase to python3

Before merging master, this PR ran `cea test --workflow slow` all the way through. I'm busy running it again just to see if that changed.

To test:

- use `conda env create` (yes, with the default `environment.yml`) to create the `cea` environment and then run the CEA from there
- also test the GUI
- we should also compare to running the workflows in the python2 versions and comparing the results.

(I'm not done testing yet - but some help would be greatly appreciated!!)


**UPDATE**: Alternatively, you can try out the [installer for v3.5.0a2](https://www.dropbox.com/s/w7toxs0mr0lr4qm/Setup_CityEnergyAnalyst_3.5.0a2.exe?dl=1) which installs the whole thing and provides the familiar CEA Console and GUI experience... (maybe install to a separate directory so you can have a side-by-side comparison?)
